intbot/core/migrations/0006_add_pretix_data_model.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 5.1.4 on 2025-04-24 22:08
+
+import uuid
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0005_add_pretalx_data_model'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PretixData',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('uuid', models.UUIDField(default=uuid.uuid4)),
+                ('resource', models.CharField(choices=[('orders', 'Orders'), ('products', 'Products'), ('vouchers', 'Vouchers')], max_length=255)),
+                ('content', models.JSONField()),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('modified_at', models.DateTimeField(auto_now=True)),
+                ('processed_at', models.DateTimeField(blank=True, null=True)),
```

Comment:
I know that we don't have any reads in this PR, but I guess the future pattern will be to read all Pretix data filtered by orders and sorted by latest? In that case it might be useful to add indexes.

Reply:
Good point! I will leave it as-is for now, but we should revisit it if the things the get slow at some point.
In practice I'm assuming we will be deleting the old data at some point, so the table will not grow that long, and it will always have  a couple hundred rows or so total – maybe up to 1-2k.